app/models/consent/default.rb

```diff
@@ -0,0 +1,65 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Consent::Default
+  def initialize(client:)
+    @client = client
+  end
+
+  def no_release_string
+    'None on file'
+  end
+
+  def revoked_consent_string
+    ''
+  end
+
+  def full_consent_string
+    ''
+  end
```

Comment:
What's the difference between `full_consent_string` and `full_release_string`?  I _think_ those should be two words for the same thing.

Reply:
I believe we can merge these into one, I had initially kept them separated as the full_consent_string was new in the implied model, but they ended up being the same thing in that class. I'll do some cleanup with these.